### DISCUSSION TITLE
상품수정 요청시 권한오류 에러가 발생됩니다.

### DISCUSSION THREAD

**Discussion by tylor79** [commerce-api-naver/commerce-solutions#56] [묻고 답하기]
안녕하세요.
솔루션마켓 테스트 중입니다.
api 사용그룹 "상품" 사용으로 되어 있습니다.
원상품수정 "PUT" /v2/products/origin-products/{originProductNo} 요청시
{
  "code": "GW.AUTHN",
  "message": "요청을 보낼 권한이 없습니다.",
  "timestamp": "2024-11-12T15:35:55.131+09:00",
  "traceId": "pr1-212535-76rf7h^1730723218721^83359732"
}
권한오류가 발생됩니다.

참고로
원상품조회  "GET" /v2/products/origin-products/{originProductNo}
정상적으로 데이트 파싱됩니다.

토큰은 정상발급 되어 헤더에 포함되어 있습니다.
토큰 account_uid로 발생 하였습니다.

        access_token = self.get_solution_auth_token()
        url = f"{self.base_url_v2}/products/origin-products/{XXXXXXXXX}"

        headers = {
            'Authorization': f"Bearer {access_token}",
            'content-Type': 'application/json'
        }

        try:
            response = requests.put(url, json=payload, headers=headers)

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

알려주신 호출건은 HTTP 요청 헤더 내 Authorization 헤더가 포함되지 않은 상태로 호출되었습니다.
HTTP Method 역시 (PUT이 아닌) GET 으로 여러 정황상 웹 브라우저에서 직접 HTTP Request가 발생한 것으로 보입니다.

API 호출에 따른 HTTP Request가 서버 사이드에서 처리되었는지 확인해보시기 바랍니다.

**Reply by tylor79** [reply]
감사합니다. 해결 되었습니다.^0^
